server/notification-providers/signl4.js

```diff
@@ -0,0 +1,60 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP, DOWN } = require("../../src/util");
+
+class SIGNL4 extends NotificationProvider {
+
+    name = "SIGNL4";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        try {
+            let data = {
+                heartbeat: heartbeatJSON,
+                monitor: monitorJSON,
+                msg,
+            };
+            // Source system
+            data["X-S4-SourceSystem"] = "UptimeKuma";
```

Comment:
I ment the following
```suggestion
                "X-S4-SourceSystem": "UptimeKuma";
            };
```

Reply:
Interesting, I always though this won't work because of the '-' in the parameter name. But it works. Will be changed. Thanks.